### PR TITLE
Eric arduino due datasheet

### PR DESCRIPTION
**Pull request by Eric286**
## What This PR Changes
- Added the firt revision of the Due Datasheet

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jhansson-ard** [comment]
Justa  note @Eric286, we've solved the issue with the spellchecker, just update your branch with main and you should be good to continue flawlessly with your PR-work ⭐

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Review done

Pending little tasks:
- Final Block Diagram
- Final Power Tree

**Review by jcarolinares — APPROVED** [review]

**Comment by jcarolinares** [comment]
Validade content linter not valid, but I need to gain control of the content to solve it, therefore I will do a legal swuash and merge to a new branch so I can fix it

File: content/hardware/10.mega/boards/due/downloads/A000056-cad-files.zip
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/downloads/A000056-cad-files.zip:1]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 1]
```diff
@@ -0,0 +1,816 @@
...
+identifier: A000062
+title: Arduino® Due
+type: maker
+author: Eric Emmanuel Chavez
+---
+
+
+<!---
+Feature Image: A descriptive image of the product to identify it on the first page of the Datasheet. Image must be put inside the "assets" folder and be named "featured.png" Actual Image for example purposes.
+-->
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:11]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 2]
```diff
@@ -0,0 +1,816 @@
...
+![](assets/featured.png)
+
+# Description
+
+<!---
+Description: A paragraph describing the product in an official and commercial way. This text is normally taken from the official description inside the Product Sales brief slides and adapted the text if necessary to the datasheet. The actual text is for examples purposes.
+-->
+
+The Arduino Due is a groundbreaking microcontroller board featuring the Atmel SAM3X8E ARM Cortex-M3 CPU, making it the first Arduino board built around a 32-bit ARM core microcontroller. With its 54 digital input/output pins, 12 analog inputs, 4 UARTs, USB OTG capability, and 84 MHz clock, the Due offers enhanced performance and versatility for a wide range of projects. Compatible with all Arduino shields designed for operation at 3.3V and compliant with the 1.0 Arduino pinout standard, the Due is a powerful tool for both beginners and experienced makers alike.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:20]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 3]
```diff
@@ -0,0 +1,816 @@
...
+
+**Warning: Unlike most Arduino boards, the Arduino Due board runs at 3.3V. The maximum voltage that the I/O pins can tolerate is 3.3V. Applying voltages higher than 3.3V to any I/O pin could damage the board.**
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:22]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 4]
```diff
@@ -0,0 +1,816 @@
...
+
+- **Robotics:** The Arduino Due can serve as the brain of an autonomous mobile robot capable of navigating and interacting with its environment. By integrating sensors such as ultrasonic range finders, gyroscopes, and encoders, developers can equip the robot with perception capabilities to sense its surroundings and detect obstacles. Using the Due's abundant I/O pins and powerful processing capabilities, algorithms for localization, mapping, and path planning can be implemented to enable autonomous navigation. Additionally, actuators such as motors or servos can be controlled by the Due to execute motion commands, allowing the robot to move and manipulate objects in its environment autonomously.
+
+- **3D Printing & CNC Machines:** the Arduino Due can function as a versatile controller for DIY projects. By interfacing stepper motor drivers and endstop switches with the Due's numerous I/O pins, enthusiasts can create their own 3D printers or CNC machines. The Due's high-speed processing capabilities enable precise control of stepper motors for accurate positioning and movement.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:46]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 5]
```diff
@@ -0,0 +1,816 @@
...
+-->
+
+The Arduino Due is a versatile microcontroller board designed for a wide range of applications. Powered by the Atmel SAM3X8E ARM Cortex-M3 CPU, it offers high performance and a robust set of features, making it suitable for complex projects. The Due's 32-bit architecture provides enhanced processing capabilities compared to traditional Arduino boards. Designed with a similar form factor to the Arduino Mega, it maintains compatibility with most Arduino shields through its extensive set of I/O pins and headers. The following table summarizes the board's main features.
+
+
+| Feature               | Description                                                                          |
+| --------------------- | ------------------------------------------------------------------------------------ |
+| Microcontroller       | **Atmel SAM3X8E ARM Cortex-M3**  32-bit ARM Cortex-M3 / 84 MHzClock speed            |
+| Memory                | **SAM3X** 512 KB Flash / 96 KB SRAM (divided into two banks: 64 KB and 32 KB)        |
+| USB-to-serial         | **ATmega16U2** Connected to the SAM3X hardware UART                                  |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:70]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 6]
```diff
@@ -0,0 +1,816 @@
...
+| Digital Inputs        | Digital Inputs (x54) not 5 V compatible                                              |
+| Analog Inputs         | The Due's analog inputs pins measure from ground to a maximum value of 3.3V (x12)    |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:72]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 7]
```diff
@@ -0,0 +1,816 @@
...
+
+
+| Feature               | Description                                                                          |
+| --------------------- | ------------------------------------------------------------------------------------ |
+| Microcontroller       | **Atmel SAM3X8E ARM Cortex-M3**  32-bit ARM Cortex-M3 / 84 MHzClock speed            |
+| Memory                | **SAM3X** 512 KB Flash / 96 KB SRAM (divided into two banks: 64 KB and 32 KB)        |
+| USB-to-serial         | **ATmega16U2** Connected to the SAM3X hardware UART                                  |
+| Digital Inputs        | Digital Inputs (x54) not 5 V compatible                                              |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:71]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 8]
```diff
@@ -0,0 +1,816 @@
...
+| Analog Inputs         | The Due's analog inputs pins measure from ground to a maximum value of 3.3V (x12)    |
+| PWM Pins              | PWM Pins with 8 bits resolution (x12)                                                |
+| Communication         | UART (x4), I2C (x2), SPI (x1 SPI header), Native USB port, Programming USB port      |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:74]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 9]
```diff
@@ -0,0 +1,816 @@
...
+| Power                 | Input voltage (VIN): 7-12 V / DC Current per I/O Pin: 8 mA                           |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:75]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 10]
```diff
@@ -0,0 +1,816 @@
...
+
+<!---
+Subsection about a Main Feature: The subsection inside features and under General Specification Overview talks about some of the main features of the product, from the microprocessor's main specifications, to the wifi chipset, Camera, Audio, Important Interfaces like CAN, etc. Ideally those sections should contain an initial 2/3 line paragraph, a table or multiple tables explaining the main specs and related media like images or schemas to explain better the technical specifications. Warnings and Note blocks regarding special things of the feature to take into account are also recommended.
+-->
+
+| Characteristics               | Details                                        |
+| ----------------------------- | ---------------------------------------------- |
+| Number of inputs              | 54x digital input, 12x analog inputs           |
+| Inputs overvoltage protection | yes                                            |
+| Antipolarity protection       | yes                                            |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:105]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 11]
```diff
@@ -0,0 +1,816 @@
...
+
+| Symbol          | Description                      | Min                | Typ | Max                | Unit |
+|-----------------|----------------------------------|--------------------|-----|--------------------|------|
+| V<sub>IN</sub>  | Input voltage from VIN pad       | 6                  | 7.0 | 16                 | V    |
+| V<sub>USB</sub> | Input voltage from USB connector | 4.8                | 5.0 | 5.5                | V    |
+| V<sub>DD</sub>  | Input high-level voltage         | 0.7*V<sub>DD</sub> |     | V<sub>DD</sub>     | V    |
+| V<sub>IL</sub>  | Input low-level voltage          | 0                  |     | 0.3*V<sub>DD</sub> | V    |
+| T<sub>OP</sub>  | Operating Temperature            | -40                | 25  | 85                 | °C   |
+
+**Note:** V<sub>DD</sub> controls the logic level and is connected to the 3.3V power rail. V<sub>AREF</sub> is for the analog logic.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:156]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 12]
```diff
@@ -0,0 +1,816 @@
...
+### Power Specification
+
+<!---
+Recommended Operating Conditions/Power Specifications: This section contains the minimum, typical and Maximum main values, normally related to voltage, current and temperature, but there can be additional specifications depending on the product. Table for example purposes. Warnings or Notes are recommended in case the product needs them to make sure the user does not commit mistakes.
+-->
+
+| Property               | Min   | Typ | Max  | Unit |
+|----------------------- |------ |-----|------|------|
+| Supply voltage         | 7     | -   | 12   | V    |
+| Permissible range      | 6     | -   | 16   | V    |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:167]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 13]
```diff
@@ -0,0 +1,816 @@
...
+Functional Overview: The functional overview contains important information about the product in terms of pinout, block diagram, mechanical information, power tree and much more. Sections and text for example purposes.
+-->
+
+### Pinout
+
+<!---
+Pinout: It is always interesting to show the pinout of the product inside the datasheet. Pinout and text for example purposes.
+-->
+
+The Arduino due pinout is shown in the following figure.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:194]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 14]
```diff
@@ -0,0 +1,816 @@
...
+
+![Arduino due pinout](assets/ad-pinout.png)
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:196]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 15]
```diff
@@ -0,0 +1,816 @@
...
+
+<!---
+Notes and warnings: If the product has some important tips to take into account regarding its pins it can be a good idea to put that information as a warning. Note code for example purposes
+-->
+
+<div style="background-color: #FFCCCC; border-left: 6px solid #FF0000; margin: 20px 0; padding: 15px;">
+<strong>Safety Note:</strong> Disconnect power before board modifications, to avoid short-circuiting.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:203]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 16]
```diff
@@ -0,0 +1,816 @@
...
+
+### Full Pinout Table
+
+<!---
+Full Pinout Table: The full pinout tables are done only necessary in case of really complex products intended for the professional sector. These tables are extracted from the Pinout Table Spreadsheet normally done before the graphical simple pinout and used as an internal tool. Following text and tables for example purposes.
+-->
+
+The full pinout of the Arduino due is available in the following tables.
+
+#### 24-Pin Header
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:214]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 17]
```diff
@@ -0,0 +1,816 @@
...
+
+<div style="text-align:center;">
+
+| Pin  | Function   | Type      | Description                                       |
+| ---- | ---------- | --------- | ------------------------------------------------- |
+|  1   | NC         | NC        | Not Connected                                     |
+|  2   | IOREF      | IOREF     | Reference for digital logic V - connected to 3.3V |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:221]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 18]
```diff
@@ -0,0 +1,816 @@
...
+|  11  | A2         | Analog    | Analog input 2 /GPIO                              |
+|  12  | A3         | Analog    | Analog input 3 /GPIO                              |
+|  13  | A4         | Analog    | Analog input 4 /GPIO                              |
+|  14  | A5         | Analog    | Analog input 5 /GPIO                              |
+|  15  | A6         | Analog    | Analog input 6 /GPIO                              |
+|  16  | A7         | Analog    | Analog input 7 /GPIO                              |
+|  17  | A8         | Analog    | Analog input 8 /GPIO                              |
+|  18  | A9         | Analog    | Analog input 9 /GPIO                              |
+|  19  | A10        | Analog    | Analog input 10 /GPIO                             |
+|  20  | A11        | Analog    | Analog input 11 /GPIO                             |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:239]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 19]
```diff
@@ -0,0 +1,816 @@
...
+|  21  | DAC0       | Analog    | Digital to Analog Converter 0                     |
+|  22  | DAC1       | Analog    | Digital to Analog Converter 1                     |
+|  23  | CANRX      | Digital   | CAN Bus Receive                                   |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:242]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 20]
```diff
@@ -0,0 +1,816 @@
...
+|  24  | CANTX      | Digital   | CAN Bus Transfer                                  |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:243]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 21]
```diff
@@ -0,0 +1,816 @@
...
+
+<caption>24-Pin Header pinout</caption>
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:245]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 22]
```diff
@@ -0,0 +1,816 @@
...
+
+</div>
+
+<!---
+Div page break tags: Sometimes the format is not perfect and some sections are added/cut in the pages we do not want to. This kind of div page break tag can be used to fix those problems
+-->
+
+#### 26-Pin Header
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:253]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 23]
```diff
@@ -0,0 +1,816 @@
...
+| 19  | D14/TX3  | Digital | GPIO 14 / Serial 3 Transmitter                  |
+| 20  | D15/RX3  | Digital | GPIO 15 / Serial 3 Receiver                     |
+| 21  | D16/TX2  | Digital | GPIO 16 / Serial 2 Transmitter                  |
+| 22  | D17/RX2  | Digital | GPIO 17 / Serial 2 Receiver                     |
+| 23  | D18/TX1  | Digital | GPIO 18 / Serial 1 Transmitter                  |
+| 24  | D19/RX1  | Digital | GPIO 19 / Serial 1 Receiver                     |
+| 25  | D20/SDA  | Digital | GPIO 20 / I2C 0 Dataline                        |
+| 26  | D21/SCL  | Digital | GPIO 21 / I2C 0 Clock                           |
+
+<caption>26-Pin Header pinout</caption>
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:286]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 24]
```diff
@@ -0,0 +1,816 @@
...
+
+</div>
+
+#### SPI
+
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:291]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 25]
```diff
@@ -0,0 +1,816 @@
...
+| 16  | D51      | Digital | GPIO 51        |
+| 17  | D53      | Digital | GPIO 53        |
+| 18  | GND      | Power   | Ground         |
+
+<caption>D22 - D53 RHS pinout</caption>
+
+</div>
+
+#### JTAG Pins
+
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:366]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 26]
```diff
@@ -0,0 +1,816 @@
...
+| 3   | TDI      | Digital | Test Data In     |
+| 4   | n/c      |    -    | Not Connected    |
+| 5   | TDO      | Digital | Test Data Out    |
+| 6   | GND      | Power   | GROUND           |
+| 7   | TCK      | Digital | Test Clock       |
+| 8   | GND      | Power   | GROUND           |
+| 9   | TMS      | Digital | Test Mode Select |
+| 10  | +3V3     | Power   | +3V3 Power Rail  |
+
+<caption>JTAG pinout</caption>
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:382]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 27]
```diff
@@ -0,0 +1,816 @@
...
+### Power Supply
+
+<!---
+Power tree: The power tree is really important from the technical point of view and for certification purposes. Must be good and consistent with Arduino`s visual style. The raw diagram is created by the R&D team and adapted and converted to the final one by the content team using Figma. Diagram and text for example purposes.
+-->
+
+The Arduino Due can be powered through multiple interfaces:
+
+- USB Type-B port (Native port and Programming port)
+- The Due can also be powered using an external voltage source connected to the VIN pin, which has a recommended voltage range of 7 to 12 V.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:405]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 28]
```diff
@@ -0,0 +1,816 @@
...
+- The Power Jack: alternatively, the Due can be powered using a DC power supply connected to the power jack, which accepts a voltage range of 7 to 12 V.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:406]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 29]
```diff
@@ -0,0 +1,816 @@
...
+
+**It is essential to note that the Arduino Due operates at 3.3 V, so any external power sources must be regulated to this voltage level. Additionally, the power supply should be able to provide sufficient current for the board's operation and any connected peripherals.**
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:408]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 30]
```diff
@@ -0,0 +1,816 @@
...
+<p style="text-align: justify;">Debugging capabilities are integrated directly into the Arduino due and are accessible via the 6-pin JTAG connector.</p>
+
+<div style="text-align:center;">
+
+| Pin | Function | Type    | Description      |
+|-----|----------|---------|------------------|
+| 1   | Reset    | Reset   | Reset            |
+| 2   | GND      | Power   | GROUND           |
+| 3   | TDI      | Digital | Test Data In     |
+| 4   | n/c      |    -    | Not Connected    |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:438]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 31]
```diff
@@ -0,0 +1,816 @@
...
+
+<p style="text-align: justify;">The Arduino Due's Native USB port features a USB Type-B connector. This port allows the board to communicate directly with a computer as a USB device, enabling functionalities such as USB host/device capabilities and USB OTG (On-The-Go) functionality.</p>
+
+#### Programming USB port
+
+<p style="text-align: justify;">This port allows the board to be programmed and powered via a USB connection to a computer. It facilitates serial communication between the Arduino Due and the computer, enabling the uploading of sketches and interaction with the Arduino IDE. The port is connected to the ATmega16U2 microcontroller, which acts as a USB-to-serial converter, simplifying the programming process. When connected to a computer, the Arduino IDE recognizes the board as a COM port, enabling seamless communication for programming and debugging purposes.</p>
+
+![Arduino Due USB port](assets/DueUSBPorts.jpg)
+
+#### 24-Pin Header Connector
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:460]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 32]
```diff
@@ -0,0 +1,816 @@
...
+
+<p style="text-align: justify;">The 24-pin header connector provides a range of interfaces and general-purpose pins essential for various applications</p>
+
+These pins offer a range of functionalities, including analog and digital input/output, power supply connections, analog-to-digital and digital-to-analog conversion, and CAN bus communication.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:464]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 33]
```diff
@@ -0,0 +1,816 @@
...
+|  11  | A2         | Analog    | Analog input 2 /GPIO                              |
+|  12  | A3         | Analog    | Analog input 3 /GPIO                              |
+|  13  | A4         | Analog    | Analog input 4 /GPIO                              |
+|  14  | A5         | Analog    | Analog input 5 /GPIO                              |
+|  15  | A6         | Analog    | Analog input 6 /GPIO                              |
+|  16  | A7         | Analog    | Analog input 7 /GPIO                              |
+|  17  | A8         | Analog    | Analog input 8 /GPIO                              |
+|  18  | A9         | Analog    | Analog input 9 /GPIO                              |
+|  19  | A10        | Analog    | Analog input 10 /GPIO                             |
+|  20  | A11        | Analog    | Analog input 11 /GPIO                             |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:489]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 34]
```diff
@@ -0,0 +1,816 @@
...
+<p style="text-align: justify;">The 24-pin header connector provides a range of interfaces and general-purpose pins essential for various applications</p>
+
+These pins offer a range of functionalities, including analog and digital input/output, power supply connections, analog-to-digital and digital-to-analog conversion, and CAN bus communication.
+
+<div style="text-align:center;">
+
+| Pin  | Function   | Type      | Description                                       |
+| ---- | ---------- | --------- | ------------------------------------------------- |
+|  1   | NC         | NC        | Not Connected                                     |
+|  2   | IOREF      | IOREF     | Reference for digital logic V - connected to 3.3V |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:471]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 35]
```diff
@@ -0,0 +1,816 @@
...
+|  15  | A6         | Analog    | Analog input 6 /GPIO                              |
+|  16  | A7         | Analog    | Analog input 7 /GPIO                              |
+|  17  | A8         | Analog    | Analog input 8 /GPIO                              |
+|  18  | A9         | Analog    | Analog input 9 /GPIO                              |
+|  19  | A10        | Analog    | Analog input 10 /GPIO                             |
+|  20  | A11        | Analog    | Analog input 11 /GPIO                             |
+|  21  | DAC0       | Analog    | Digital to Analog Converter 0                     |
+|  22  | DAC1       | Analog    | Digital to Analog Converter 1                     |
+|  23  | CANRX      | Digital   | CAN Bus Receive                                   |
+|  24  | CANTX      | Digital   | CAN Bus Transfer                                  |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:493]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 36]
```diff
@@ -0,0 +1,816 @@
...
+
+<caption>24-Pin Header pinout</caption>
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:495]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 37]
```diff
@@ -0,0 +1,816 @@
...
+
+</div>
+
+#### 26-Pin Header Connector
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:499]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 38]
```diff
@@ -0,0 +1,816 @@
...
+| 19  | D14/TX3  | Digital | GPIO 14 / Serial 3 Transmitter                  |
+| 20  | D15/RX3  | Digital | GPIO 15 / Serial 3 Receiver                     |
+| 21  | D16/TX2  | Digital | GPIO 16 / Serial 2 Transmitter                  |
+| 22  | D17/RX2  | Digital | GPIO 17 / Serial 2 Receiver                     |
+| 23  | D18/TX1  | Digital | GPIO 18 / Serial 1 Transmitter                  |
+| 24  | D19/RX1  | Digital | GPIO 19 / Serial 1 Receiver                     |
+| 25  | D20/SDA  | Digital | GPIO 20 / I2C 0 Dataline                        |
+| 26  | D21/SCL  | Digital | GPIO 21 / I2C 0 Clock                           |
+
+<caption>26-Pin Header pinout</caption>
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:536]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 39]
```diff
@@ -0,0 +1,816 @@
...
+#### SPI
+
+<p style="text-align: justify;">These pins facilitate communication between the Arduino Due and external SPI devices </p>
+
+<div style="text-align:center;">
+
+| Pin | Function | Type     | Description                  |
+|-----|----------|----------|------------------------------|
+| 1   | CIPO     | Internal | Controller In Peripheral Out |
+| 2   | +5V      | Internal | Power Supply of 5V           |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:549]: DONE

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 40]
```diff
@@ -0,0 +1,816 @@
...
+All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB, it is possible to enter bootloader mode by double-tapping the reset button right after the power-up.
+
+## Mechanical Information
+
+<!---
+Mechanical Information: Information related to measurements and drawings, putting special attention to the main components, connectors, holes and more. Nano Matter drawings and text as an example.
+-->
+
+<p style="text-align: justify;">
+The Arduino Due is a microcontroller board measuring 101.52 mm x 53.3 mm, featuring two USB-B connectors.</p>
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:656]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 41]
```diff
@@ -0,0 +1,816 @@
...
+
+### Board Dimensions
+
+The Arduino Due board outline and mounting holes dimensions are shown in the figure below; all the dimensions are in mm. 
+
+![Arduino Due Mounting Holes And Board Outline](assets/dueMechanical.png)
+
+### Board Connectors
+
+Connectors of the Arduino Due are placed on the left side of the board; their placement is shown in the figure below; all the dimensions are in mm.
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:666]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 42]
```diff
@@ -0,0 +1,816 @@
...
+|----------------------------------------|-------------------------|
+| Lead (Pb)                              | 1000                    |
+| 02/11/2023                             | 2                       |
+| 25/10/2023                             | 1                       |
+| Poly Brominated Biphenyls (PBB)        | 1000                    |
+| Poly Brominated Diphenyl ethers (PBDE) | 1000                    |
+| Bis(2-Ethylhexyl) phthalate (DEHP)     | 1000                    |
+| Benzyl butyl phthalate (BBP)           | 1000                    |
+| Dibutyl phthalate (DBP)                | 1000                    |
+| Diisobutyl phthalate (DIBP)            | 1000                    |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:713]: done

File: content/hardware/10.mega/boards/due/datasheet/datasheet.md [hunk 43]
```diff
@@ -0,0 +1,816 @@
...
+Certification Summary: A table including the main certification of the products. Add certifications serial names if possible.
+-->
+
+| **Certification** | **Status** |
+|:-----------------:|:----------:|
+|  CE/RED (Europe)  |    Yes     |
+|     UKCA (UK)     |    Yes     |
+|     FCC (USA)     |    Yes     |
+|    IC (Canada)    |    Yes     |
+| MIC/Telec (Japan) |    No      |
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/10.mega/boards/due/datasheet/datasheet.md:689]: done

**Pull request merged**